modules/districtmDmxAdapter.js

```diff
@@ -0,0 +1,158 @@
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+import {config} from 'src/config';
+
+const BIDDER_CODE = 'districtmDMX';
+
+const DMXURI = 'https://dmx.districtm.io/b/v1';
+
+export class DistrictmDmxAdapter {
```

Comment:
Please use pattern described here http://prebid.org/dev-docs/bidder-adaptor.html 
Prebid is not using `class` as of now. This will transpile fine and i think will also work in all browsers.